cruize_operations/views.py

```diff
@@ -1,36 +1,83 @@
 from rest_framework import status
 from rest_framework.response import Response
 from rest_framework.viewsets import GenericViewSet
+
+from components import FirebaseDataManager
 from cruize_operations import (
     RepayToAaveRequestSerializer,
     CruizeDepositRequestSerializer,
+    FirebaeRequestSerializer,
+    FirebaseFecthRequestSerializer,
 )
 
+
 from services.contracts.cruize.cruize_contract import Cruize
+from utilities import cruize_constants
 
 
 class CruizeOperations(GenericViewSet):
+    def initialize(self):
+        self.cruize_contract_ref = Cruize()
```

Comment:
Oh, if def initialize is just initializing 1 class object then we shouldn't keep it.
`self.cruize_contract_ref = Cruize()` can be written in all api functions.